src/shared/ExternalLink/ExternalLink.tsx

```diff
@@ -0,0 +1,28 @@
+import { Button, ButtonProps, ButtonVariant } from "@patternfly/react-core";
+import { ExternalLinkAltIcon } from "@patternfly/react-icons/dist/js/icons/external-link-alt-icon";
+import { FunctionComponent } from "react";
+
+export type ExternaLinkProps = {
+  testId: string;
+  target?: ButtonProps["target"];
+  href: NonNullable<ButtonProps["href"]>;
+};
+
+export const ExternalLink: FunctionComponent<ExternaLinkProps> = ({
+  testId,
+  target = "_blank",
+  href,
+  children,
+}) => (
+  <Button
+    data-testid={testId}
+    isInline
+    variant={ButtonVariant.link}
+    component="a"
+    target={target}
+    href={href}
+  >
+    {children}
+    <ExternalLinkAltIcon className="pf-u-ml-xs" />
```

Comment:
I really like that you created a shared component for this. Something I noticed when we were using this pattern for the Overview page was that depending on the size of the viewport, the icon might wrap to the next line. This could be potentially confusing where it's less obvious that the link icon is meant to describe the behavior of the link, because they're not adjacent in that case.

Below is an example of this component showing that the icon wraps. I realize this is an edge case, and am not sure if there's a way to solve that (e.g. can we use `&nbsp` between the icon and text here?), but wanted to just mention this in case it's an easy thing to address. 

![image](https://user-images.githubusercontent.com/21063328/167951907-87b6f292-4b23-4e8b-a451-e3a5b412897f.png)

Reply:
Nice catch! I think this should not happen anymore now :)